testinfra/modules/pip.py

```diff
@@ -100,11 +130,40 @@ def get_outdated_packages(self, pip_path="pip"):
                 re.compile(r"^(.+?) \((.+)\) - Latest: (.+) .*$"),
                 re.compile(r"^(.+?) \(Current: (.+) Latest: (.+) .*$"),
             ]
-            for line in self.check_output("{0} list -o".format(pip_path)).splitlines():
+            for line in cls.check_output("{0} list -o".format(pip_path)).splitlines():
                 if line.startswith("Warning: "):
                     # Warning: cannot find svn location for ...
                     continue
                 output_re = regexpes[1] if "Current:" in line else regexpes[0]
                 name, current, latest = _re_match(line, output_re)
                 pkgs[name] = {"current": current, "latest": latest}
         return pkgs
+
+
+class PipPackage(Pip):
+    """.. deprecated:: 6.2
+
+    Use :class:`~testinfra.modules.pip.Pip` instead.
+    """
+
+    @staticmethod
+    def _deprecated():
+        warnings.warn(
+            "Calling host.pip_package is deprecated, call host.pip instead",
+            DeprecationWarning,
+        )
+
+    @classmethod
+    def check(cls, pip_path="pip"):
+        PipPackage._deprecated()
+        return super(PipPackage, cls).check(pip_path=pip_path)
```

Comment:
We are python3 only so you can write `` return super().check(pip_path=pip_path)`` here and in following code.

Reply:
Right, thanks, it's done.